### PR TITLE
Add Vulnerability-detector new configuration

### PR DESCRIPTION
**Pull request by 0xN3utr0n**
## Description 
HI team,
The purpose of this PR is to modify the current configuration for RedHat and Debian, and include new feeds. Special attention has been taken for RedHat, since the new configuration is quite complex.

Kind regards,
Fernando

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns.

### CODE REVIEW

File: source/user-manual/capabilities/vulnerability-detection/offline_update.rst [hunk 1]
```diff
@@ -35,11 +35,11 @@ To fetch the vulnerability feeds from an alternative repository, the configurati
 .. code-block:: xml
 
     <provider name="canonical">
-        <enabled>no</enabled>
-        <os path="/local_path/com.ubuntu.focal.cve.oval.xml">focal</os>
-        <os path="/local_path/com.ubuntu.bionic.cve.oval.xml">bionic</os>
-        <os path="/local_path/com.ubuntu.xenial.cve.oval.xml">xenial</os>
-        <os path="/local_path/com.ubuntu.trusty.cve.oval.xml">trusty</os>
+        <enabled>yes</enabled>
+        <os url="http://local_repo/com.ubuntu.focal.cve.oval.xml">focal</os>
```
Comment (chemamartinez) [inline source/user-manual/capabilities/vulnerability-detection/offline_update.rst:39]: Ubuntu feeds fetched from a remote URL should be compressed:
```xml
<os url="http://local_repo/com.ubuntu.focal.cve.oval.xml">focal</os>
```

File: source/user-manual/capabilities/vulnerability-detection/offline_update.rst [hunk 2]
```diff
@@ -102,10 +106,89 @@ In order to update the vulnerability feeds from an alternative repository, the c
         <update_interval>1h</update_interval>
     </provider>
 
+JSON feed
```
Comment (chemamartinez) [inline source/user-manual/capabilities/vulnerability-detection/offline_update.rst:109]: Call it `Debian Security Tracker JSON` or something similar

File: source/user-manual/capabilities/vulnerability-detection/offline_update.rst [hunk 3]
```diff
@@ -102,10 +106,89 @@ In order to update the vulnerability feeds from an alternative repository, the c
...
+^^^^^^^^^
+To perform an offline update of the Debian JSON feed, you must download the corresponding file:
```
Comment (chemamartinez) [inline source/user-manual/capabilities/vulnerability-detection/offline_update.rst:111]: Can you explain here what is the data collected from this feed?

File: source/user-manual/capabilities/vulnerability-detection/offline_update.rst [hunk 4]
```diff
@@ -102,10 +106,89 @@ In order to update the vulnerability feeds from an alternative repository, the c
...
+    <provider name="debian">
+        <enabled>yes</enabled>
+        <url>/http://local_repo/security_tracker_local.json</url>
+        <update_interval>1h</update_interval>
+    </provider>
+
 Red Hat
 ^^^^^^^
+.. note:: 
+    For Red Hat we use two different feeds (OVAL and JSON), and for each one of them the offline update must be explicitly indicated; otherwise, the default mode will be assumed.
```
Comment (chemamartinez) [inline source/user-manual/capabilities/vulnerability-detection/offline_update.rst:142]: Same as for Debian, add a simple explanation of what data is collected from each source

File: source/user-manual/capabilities/vulnerability-detection/offline_update.rst [hunk 5]
```diff
@@ -51,18 +51,24 @@ Alternatively, the feeds can be loaded from a local path. To achieve it, the ``p
...
+        <os path="/local_path/com.ubuntu.focal.cve.oval.xml">focal</os>
+        <os path="/local_path/com.ubuntu.bionic.cve.oval.xml">bionic</os>
+        <os path="/local_path/com.ubuntu.xenial.cve.oval.xml">xenial</os>
+        <os path="/local_path/com.ubuntu.trusty.cve.oval.xml">trusty</os>
         <update_interval>1h</update_interval>
     </provider>
 
 Debian
 ^^^^^^
+The current implementation of Debian requires two different feeds at runtime in order to achieve the highest precision possible.
```
Comment (chemamartinez) [inline source/user-manual/capabilities/vulnerability-detection/offline_update.rst:64]: Change this introduction and the note, same for the one for RedHat:

Currently, the Debian vulnerabilities are fetched from two different sources. One OVAL file for each supported Debian version, and a JSON feed with global information of the affected packages for each distribution. 

Both sources are necessary for the proper functioning of the scanner, how to configure each of them for an offline update is described below.

**Review by chemamartinez — APPROVED** [review]
LGTM!

**Review by davidfmiranda — APPROVED** [review]
LGTM

**Pull request merged**